source/help/apps/desktop-guide.rst

```diff
@@ -1,104 +1,95 @@
-===================================
 Using Mattermost Desktop App
-===================================
+============================
 
-Mattermost desktop applications are available for Windows, Mac and Linux operating systems. 
+Mattermost desktop applications are available for Windows, Mac and Linux operating systems.
 
-You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during set up and for troubleshooting tips.
+You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during setup and for troubleshooting tips.
 
 To view the latest updates, please see our `changelog <https://docs.mattermost.com/help/apps/desktop-changelog.html>`_.
 
 Below are a few tips to get you started and to configure your experience on the desktop app:
 
- - `Team Management <https://docs.mattermost.com/help/apps/desktop-guide.html#id1>`_
- - `App Options <https://docs.mattermost.com/help/apps/desktop-guide.html#id2>`_
- - `Menu Bar <https://docs.mattermost.com/help/apps/desktop-guide.html#id3>`_
+.. contents::
+    :backlinks: top
+    :local:
 
-Team Management
----------------------------------------------------------------------
+Server Management
+-----------------
 
 You can connect to multiple Mattermost servers from a single interface on the desktop app.
 
-The Team Management section allows you to add, edit and remove servers and teams.
+The server Management section allows you to add, edit and remove servers.
 
-Adding servers and teams
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Adding servers
+~~~~~~~~~~~~~~
 
-To add a new server on the Mattermost instance, 
+To add a new server on the Mattermost instance:
 
-1. On the menu bar, go to **File** > **Settings** or press CTRL/CMD+COMMA
-2. Click **Add new team** in the Team Management section.
-3. In the **Name** field, enter the name that you want for the tab. 
+1. On the menu bar, go to **File > Settings**.
+2. In the *Server Management* section, click **Add new server**.
+3. In the **Name** field, enter the name that you want for the tab.
 4. In the **URL** field, enter the complete URL of the server that you want to connect to. Must begin with either ``http://`` or ``https://``.
 5. Click **Add**.
-6. Click **Save**.
 
-You can now access all teams you have joined in the server once you have saved the settings, and each will appear as a separate tab in the app. 
+To access all teams that you have joined, click the name on the Server Management section. Each server appears as a separate tab in the app.
```

Comment:
Thank you! I should be more careful about the trailing whitespaces.

1. click the name of a server
![image](https://cloud.githubusercontent.com/assets/13119842/23325653/e800987c-fac4-11e6-8cbe-fa8ae63fb23c.png)
2. "in" is probably the correct word